src/packages/objectmanager/objectmanager.js

```diff
@@ -9,11 +9,12 @@ export class ObjectManager {
      * @returns ObjectManager
      */
     constructor() {
-        this.objects = {}
+        this.objects = new Map()
     }
 
+
     destroy() {
-        this.objects = {}
+        this.objects = new Map()
```

Comment:
Die Objekte, die vorher in der Map gespeichert wurden, haben keine explizite ``destroy`` Funktion oder?

Reply:
Nein, momentan nicht (https://github.com/joonlabs/fox.js/blob/master/src/packages/gameobjects/gameobject.js). Kann man später ergänzen.